README.md

```diff
@@ -135,13 +135,29 @@ and follows the mainly used project structure in most Spring Boot projects.
 
 This application provides container images for demonstration purposes.
 
-DockerHub: https://hub.docker.com/u/tractusx/vas-country-risk-backend
+DockerHub: [https://hub.docker.com/u/tractusx/vas-country-risk-backend](https://hub.docker.com/u/tractusx/vas-country-risk-backend)
```

Comment:
If you have a dedicated `DOCKER_NOTICE.md`, you should link to it from your `README.md` instead of duplicating the content